psc-bundle/Main.hs

```diff
@@ -109,6 +110,12 @@ options = Options <$> some inputFile
     <> long "require-path"
     <> help "The path prefix used in require() calls in the generated JavaScript [deprecated]"
 
+  shouldUncurry :: Parser String
+  shouldUncurry = strOption $
+       short 'p'
```

Comment:
`psc` currently has one option for turning each individual optimization on or off, `--no-tco`, `--no-magic-do`, with no short version so perhaps we should follow suit here? `--[no-]optimize-uncurry`?